### PR TITLE
chore: translations for ticket fields module

### PR DESCRIPTION
**Pull request by gosiexon-zen**
## Description
This PR adds the source file for the translations used in the Ticket Fields module. We are using ticket fields in two modules now so we decided to separate fields from new request form. I am targeting the master branch because our internal tooling is fetching the translations from master. So, to start the translation process, we need to push the YAML file on master even if the code is still in the feature branch: https://github.com/zendesk/copenhagen_theme/tree/service-catalog-eap

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: Changes to the UI are [tested for accessibility](./../README.md#accessibility-testing) and compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
- [ ] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [ ] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

**Comment by gosiexon-zen** [comment]
Hi @kubraokcu we moved translations for ticket fields from new request form module to separate module so the keys were changes. This was also checked already in PR's to Service Catalog feature branch but please have a second look 🙏

**Review by Fredx87 — APPROVED** [review]

File: src/modules/ticket-fields/translations/en-us.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - translation:
+      key: "cph-theme-ticket-fields.lookup-field.placeholder"
+      title: "Placeholder shown in combobox, {{label}} will be replaced by field label. Alternative English for languages with grammar/declension issues: [Search label {{label}}]."
+      screenshot: "https://drive.google.com/file/d/1nlnj_I-L7bkIs9Hn93Btrl8bh_HJfL6h/view?usp=drive_link"
+      value: "Search {{label}}"
+  - translation:
+      key: "cph-theme-ticket-fields.credit-card-digits-hint"
+      title: "Additional label shown above the credit card field, informing the user that only the last 4 digits are required."
+      screenshot: "https://drive.google.com/file/d/13Pz2p5q7FHX9FKT6MUR6Sg5vWy55X0kP/view?usp=drive_link"
+      value: "(Last 4 digits)"
```
Comment (kubraokcu) [inline src/modules/ticket-fields/translations/en-us.yml:37]: Didn't we have a variable for the credit card field name?
`{field_name} (last 4 digits)`
Here the space between the name and the suffix will be hard-coded, it's not localization-friendly for Asian and Cyrillic languages which have different space width or no space at all

Comment (gosiexon-zen) [reply src/modules/ticket-fields/translations/en-us.yml:37]: We decided to do this that way in other PR months ago here was the conversation: https://github.com/zendesk/copenhagen_theme/pull/474

Comment (kubraokcu) [reply src/modules/ticket-fields/translations/en-us.yml:37]: Ahh thank you, apologies I didn't remember about this conversation

**Review by kubraokcu — APPROVED** [review]

**Pull request merged**